indexer/services/ender/src/validators/order-fill-validator.ts

```diff
@@ -32,6 +37,30 @@ export class OrderFillValidator extends Validator<OrderFillEventV1> {
     }
   }
 
+  public async getEventForBlockProcessor(): Promise<object> {
+    if (this.event.order) {
```

Comment:
nit: add comment for why we only pull from CanceledOrdersCache if event.order is populated